.github/workflows/test.yml

```diff
@@ -4,10 +4,6 @@
 name: Tests
 
 on:
-  push:
-    branches:
-      - '**'
-
   pull_request:
     branches:
       - '**'
```

Comment:
Will this run when new commits are pushed to a PR?

Reply:
As discussed on vc it'll run when new commits are pushed to a PR since by default this workflow will run on these events : `opened, synchronize & reopened`

Docs : https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request